### PR TITLE
log to stdout, 3 backups on Sunday

### PR DESCRIPTION
**Pull request by scherniavsky**
- fixes logging to stdout that stops after the 1-st backup
- adds 1 more backup point on Sunday and aligns the Sunday schedule times with the weekly ones

### CODE REVIEW

**Comment by scherniavsky** [comment]
:+1:

File: start_backup.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ if [ ! -d "/data/ghe-production-data/" ]
 then
     mkdir -p /data/ghe-production-data/
 fi
-/backup/backup-utils/bin/ghe-backup 1>> /var/log/ghe-prod-backup.log 2>&1
+/backup/backup-utils/bin/ghe-backup 2>&1 | tee -a /var/log/ghe-prod-backup.log
```
Comment (rashamalek) [inline start_backup.sh]: I'd suggest to remove any log files, and directly log to stdout

File: Dockerfile [hunk 2]
```diff
@@ -68,7 +68,7 @@ RUN \
   chmod 0700 /start_backup.sh && \
   chmod 0700 /backup/replace-convert-properties.sh && \
   chmod 0700 /backup/final-docker-cmd.sh && \
-  mkfifo /var/log/ghe-prod-backup.log && \
+  touch /var/log/ghe-prod-backup.log && \
```
Comment (rashamalek) [inline Dockerfile]: I'd suggest to remove any log files, and directly log to stdout

**Review by rashamalek — CHANGES_REQUESTED** [review]

**Comment by scherniavsky** [comment]
:+1:

**Comment by rashamalek** [comment]
👍

**Comment by scherniavsky** [comment]
:+1:

**Comment by lars-zalando** [comment]
:+1:

**Pull request merged**